src/cephadm/tests/test_cephadm.py

```diff
@@ -2404,6 +2404,41 @@ def test_ip_in_subnet(self):
         with pytest.raises(Exception):
             rc = cd.ip_in_sublets('fe80:2030:31:24', 'fe80::/64')
 
+    @pytest.mark.parametrize("conf", [
+    """[global]
+public_network='1.1.1.0/24,2.2.2.0/24'
+cluster_network="3.3.3.0/24, 4.4.4.0/24
```

Comment:
I sort of wish that unmatched quotes could still be an error (unless you tell me it wasn't in the old version), but I won't let it bother me too much.

Reply:
while it technically would work if someone provided this with the given code, I definitely did not intend to purposely support this and have now removed this test case (by adding the ending quote that should have been there)